src/unwind_symbols.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google LLC
```

Comment:
I'm not sure whether I can. If you feel I need to, I'll can ask around internally to see whether "Copyright 2019 Tock OS Developers" would be acceptable, but that will take time.

Oh, since I forgot to mention it before: this contribution is dual-licensed under Apache 2.0 & MIT, as requested by the project's README.

Reply:
Done.